packages/terra-navigation-layout/src/terra-dev-site/doc/example/HeaderExample.jsx

```diff
@@ -1,41 +1,35 @@
 import React from 'react';
+import classNames from 'classnames/bind';
 import PropTypes from 'prop-types';
 import { Link, withRouter } from 'react-router-dom';
+import styles from './HeaderExample.scss';
+
+const cx = classNames.bind(styles);
 
 const propTypes = {
   layoutConfig: PropTypes.shape({}),
 };
 
-const inlineButtonStyle = {
-  display: 'inline', marginLeft: '5px', height: '25px', border: '1px dashed lightgrey',
-};
-
 const HeaderExample = ({ layoutConfig }) => (
-  <div style={{
-    height: '60px', width: 'calc(100% - 10px)', border: '4px dashed lightgrey', margin: '5px', position: 'relative', display: 'flex',
-  }}
-  >
-    <div style={{
-      position: 'absolute', top: '50%', left: '50%', color: 'grey', transform: 'translate3d(-50%, -50%, 0)',
-    }}
-    >
-      <h2 style={{ display: 'inline', margin: 0 }}>Header</h2>
+  <div className={cx('content-wrapper1')}>
+    <div className={cx('content-wrapper2')}>
+      <h2>Header</h2>
       {layoutConfig.toggleMenu
         && (
-        <button
-          type="button"
-          style={inlineButtonStyle}
-          onClick={layoutConfig.toggleMenu}
-        >
-Toggle Menu
-        </button>
+          <button
+            type="button"
+            onClick={layoutConfig.toggleMenu}
+            className={cx('button')}
```

Comment:
There isn't a button class name in the HeaderExampler.scss file. Do we need this?

Reply:
Fixed here https://github.com/cerner/terra-framework/pull/742/commits/d320272a142e98bfee8b1d612816f9fabd7aa36d.